packages/date-picker/src/DatePicker/DatePicker.scss

```diff
@@ -3,119 +3,130 @@
 @import "~@kaizen/design-tokens/sass/spacing";
 @import "~@kaizen/design-tokens/sass/border";
 @import "~@kaizen/design-tokens/sass/shadow";
-@import "~@kaizen/draft-button/KaizenDraft/Button/styles";
 
 // Vars
-$border-radius: 3px;
+$border-radius: 7px;
+$button-height: 48px;
+$button-base-padding-horizontal: $spacing-sm;
+$button-icon-size: 1.25rem; // 20px
+
+$disabled-opacity: 0.5;
+$placeholder-opacity: 0.7;
+
+@mixin button-reset {
+  appearance: none;
+  display: inline;
+  background: transparent;
+  color: inherit;
+  font: inherit;
+  margin: 0;
+  padding: 0;
+  border: none;
+}
 
-.calendar {
+.button {
+  @include button-reset;
+  position: relative;
+  display: flex;
+  align-items: center;
+  height: $button-height;
+  width: 100%;
+  color: $color-purple-800;
+  font-family: $typography-paragraph-body-font-family;
+  font-size: $typography-paragraph-body-font-size;
+  font-weight: $typography-paragraph-body-font-weight;
+  line-height: 1.5;
+  letter-spacing: $typography-paragraph-body-letter-spacing;
+  text-align: left;
   background-color: $color-white;
-  font-family: $typography-heading-4-font-family;
-  z-index: 1;
-
-  .wrapper {
-    padding-bottom: 0;
-  }
-
-  .navbar {
-    position: absolute;
-    display: flex;
-    justify-content: space-between;
-    width: calc(100% - 3rem);
-    margin: 0 $spacing-md;
-    top: 1.25rem;
+  background-clip: padding-box;
+  border: $border-solid-border-width $border-solid-border-style $color-gray-500;
+  border-radius: $border-radius;
+  padding: 0 $button-base-padding-horizontal;
+
+  &:focus:not([disabled]),
+  &:hover:focus:not([disabled]),
+  &:hover:not([disabled]) {
+    background-color: $color-gray-200;
+    border-color: $color-gray-600;
     color: $color-purple-800;
   }
 
-  .arrows {
-    @include button-reset;
-    display: inherit;
-    padding: 0.25rem $spacing-xs;
-    border: $border-solid-border-width $border-solid-border-style transparent;
-    border-radius: $border-radius;
-
-    &:hover {
-      cursor: pointer;
-      background-color: $color-blue-100;
-      color: $color-blue-500;
-    }
-
-    &:focus {
-      outline-offset: -2px;
-      outline: $color-blue-500 $border-focus-ring-border-style
-        $border-focus-ring-border-width;
-    }
-  }
-
-  .month {
-    margin: $spacing-md;
-    border-collapse: separate;
-    border-spacing: 1px;
+  &:focus:not([disabled]) {
```

Comment:
This can be addressed later if we want, but usually we use `focus-visible` instead of `focus` so that the border/outline only shows when navigating with a keyboard (this will show on click).

We have a polyfill that adds a `focus-visible` class to elements when they are focused using a keyboard (Safari doesn't support :focus-visible). You just need to use `:global(.focus-visible)` to target so that you target the class name as it is, instead of a CSS Module uniquified class name.

Reply:
Is this for every instance or just this line? We're using focus quite a bit in combination with hover etc.

Reply:
Every instance